src/websys_bridge.rs

```diff
@@ -388,16 +405,50 @@ pub fn el_from_ws<Ms>(node: &web_sys::Node) -> Option<El<Ms>> {
                     .get(i)
                     .expect("Can't find child in raw html element.");
 
-                if let Some(child_vdom) = el_from_ws(&child) {
+                if let Some(child_vdom) = node_from_ws(&child) {
                     result.children.push(child_vdom);
                 }
             }
-            Some(result)
+            Some(Node::Element(result))
         }
-        3 => Some(El::new_text(&node.text_content().expect("Can't find text"))),
+        3 => Some(Node::Text(Text::new(
```

Comment:
Idea:
Create enum / constants for `node_type` values.

Reply:
Since this is currently used only in this fn, and we're essentially mapping those DOM-standard numbers to `Node` variants here, I don't think it's necessary.

Reply:
**RESOLVED**